modules/sw360/sw360-workflow/src/main/java/org/eclipse/sw360/antenna/sw360/workflow/SW360ConnectionConfigurationFactory.java

```diff
@@ -72,6 +72,13 @@
      */
     public static final String CLIENT_PASSWORD_KEY = "client.password";
 
+    /**
+     * The name of the property from the Antenna tool configuration defining
+     * the token of the OAuth 2 client to be passed to the authorization
+     * server when requesting an access token.
+     */
+    public static final String TOKEN_KEY = "user.token";
```

Comment:
It would be great if this new property could be documented together with the other SW360-related properties supported by Antenna. This would have to be done in the files sw360-update-generator-step.md and sw360-enricher.md.

Reply:
updated the .md files with user.token attribute